front/docker/Dockerfile.playwright

```diff
@@ -0,0 +1,19 @@
+FROM alpine AS exec_as_builder
+
+RUN apk add build-base
+COPY front/docker/exec-as.c .
+RUN gcc -std=c99 -static -o /exec-as exec-as.c
+
+FROM mcr.microsoft.com/playwright
```

Comment:
Hm, I wonder… We're pulling the Playwright Docker image, and then we're installing Playwright inside it: do we really need both?

Reply:
I believe we can stop installing playwright with its deps, as long as we make sure to match the image we pull to the playwright version we have in the yarn.lock. This means bumping the dockerfile every time we bump our yarn playwright version, and thus more things to maintain.

I don't know which is preferable.

Reply:
A middleground could be to keep the install with-deps but still try to match the versions.

The install with-deps is three times faster (on my computer) when the versions correctly match, so keeping it ensures that the script won't crash without sacrificing too much time.

Reply:
We need to make sure to use the same Playwright version as `package.json`: we've had breakage in the past due to Playwright upgrades.

I'd like not to duplicate fixed dep versions in two places because it's so easy for them to go out of sync (especially with dependabot).

Reply:
In any cases we should end up using the correct version, the question is only if we try to match the image version to the yarn.lock to avoid installation or if we install it by hand.

If you want to match the versions, but don't want to repeat them, what would you suggest? @khoyo proposed adding a cli rule to enforce the versions matching. Would that be enough?

Reply:
If you're willing to add a dependency to jq on your script (this package is available on all distributions and systems but it generally needs to be installed manually) you can do something like this:

```
yarn list --pattern playwright --json | jq -r '.data.trees[].name | split("@")[-1]' | sort -u
```

Then you export this inside a variable and your variabilize the tag of the docker immage using a build arg:

```
ARG PLAYWRIGHT_VERSION
FROM playwright:$PLAYWRIGHT_VERSION
```

Reply:
Looks good thanks!

Reply:
The Playwright Docker image already includes the _Playwright browsers_ and all necessary _browser system dependencies_, so running `yarn playwright install --with-deps` is unnecessary, as it only installs _browser system dependencies_. 
We should always keep the Docker image updated to the latest version, as it primarily affects the browsers.

So the version of Playwright, specified in` package.json` doesn’t need to match the Docker image version. And the upgrade of this version can cause  breakages and failures. 
However,we can updated it to a stable and recent version, like v1.47.2.